### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Oleh-Rybachuk**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Frontend practice with catalog page
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_catalog/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_catalog/report/html_report/)
+- [DEMO LINK](https://oleh-rybachuk.github.io/layout_catalog/)
+- [TEST REPORT LINK](https://oleh-rybachuk.github.io/layout_catalog/report/html_report/)
```
Comment (OlEzhA0) [inline readme.md:4]: add this links to PR description

File: src/index.html [hunk 2]
```diff
@@ -4,9 +4,365 @@
...
+
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="Imac picture"
+          class="card__img"
+
```
Comment (OlEzhA0) [inline src/index.html:81]:
```suggestion
```

File: src/index.html [hunk 3]
```diff
@@ -4,9 +4,365 @@
...
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__review">Reviews: 5</p>
+        </div>
+
+        <div class="card__price-text">
+          <p class="card__price">Price:</p>
+
```
Comment (OlEzhA0) [inline src/index.html:102]:
```suggestion
```

File: src/index.html [hunk 4]
```diff
@@ -4,9 +4,365 @@
...
+          <p class="card__price-sum">$2,199</p>
+        </div>
+
+        <div
+        
+          data-qa="card-hover"
+        >
```
Comment (OlEzhA0) [inline src/index.html:109]:
```suggestion
        <div data-qa="card-hover">
```

File: src/index.html [hunk 5]
```diff
@@ -4,9 +4,365 @@
...
+          <a
+
+            class="card__button"
+            href="#"
+
+          >
```
Comment (OlEzhA0) [inline src/index.html:115]:
```suggestion
          <a class="card__button" href="#">
```

**Review by OlEzhA0 — CHANGES_REQUESTED** [review]
Hey! Please check my comments and push scss files to github

**Review by OlEzhA0 — CHANGES_REQUESTED** [review]
Hey! Please check my comments and push scss files to github

**Comment by Oleh-Rybachuk** [comment]
https://oleh-rybachuk.github.io/layout_catalog
https://oleh-rybachuk.github.io/layout_catalog/report/html_report

**Comment by Oleh-Rybachuk** [comment]
https://oleh-rybachuk.github.io/layout_catalog
https://oleh-rybachuk.github.io/layout_catalog/report/html_report

**Comment by Oleh-Rybachuk** [comment]
https://oleh-rybachuk.github.io/layout_catalog
https://oleh-rybachuk.github.io/layout_catalog/report/html_report

**Pull request closed without merging**